src/mame/drivers/xavix.cpp

```diff
@@ -129,13 +126,31 @@ class xavix_state : public driver_device
 	uint8_t m_irq_vector1_lo_data;
 	uint8_t m_irq_vector1_hi_data;
 
+	uint8_t m_multparams[3];
+	uint8_t m_multresults[2];
+
 	uint8_t get_vectors(int which, int half);
 
+	required_shared_ptr<uint8_t> spr_attr0;
+	required_shared_ptr<uint8_t> spr_attr1;
+	required_shared_ptr<uint8_t> spr_attr2;
+	required_shared_ptr<uint8_t> spr_attr3;
+
+	required_shared_ptr<uint8_t> spr_attr5;
+	required_shared_ptr<uint8_t> spr_attr6;
+	required_shared_ptr<uint8_t> spr_attr7;
```

Comment:
These don't have the `m_` prefix that other members have here.

Reply:
well they're only temporary names while I figure out what each table is anyway, but yeah, I should probably add them for the hell of it